src/Shop.js

```diff
@@ -0,0 +1,84 @@
+class Shop {
+  constructor(items=[]){
+    this.items = items;
+  }
+
+  updateQuality() {
+    for (var i = 0; i < this.items.length; i++) {
+      var item = this.items[i];
+      var name = item.name;
+      if (name.includes('Sulfuras')) {
+        continue;
+      }
+      else if (name.includes('Aged Brie')) {
```

Comment:
Can you talk about why you choose to use .includes to evaluate the items? What are the downsides?
> var name = "Special Aged Brie";
> name.includes('Aged Brie');
< true

Reply:
I see. My impression was that any kind of Aged Brie would have the special properties, so I wanted the program to be able to process all Aged Brie as such, including Special Aged Brie. However, if the specs of only exactly "Aged Brie" are to be applied, I could use:

else if (name == 'Aged Brie') {

Reply:
Yeah, there is definitely no way to know what new logic will be required as the application evolves. I just wanted to bring it up but it seems you are aware of the differences. For this case, you can leave it as .includes as your intention was to process them the same way. As long as you understand implications of your decisions and your code is maintainable.